ui/StatusQ/src/StatusQ/Core/Theme/ThemePalette.qml

```diff
@@ -246,6 +248,13 @@ QtObject {
         property color menuItemHoeverBackgroundColor
     }
 
+    property QtObject statusMessage: QtObject {
+        property color emojiReactionBackground
+        property color emojiReactionBackgroundHovered
+        property color emojiReactionActiveBackground
+        property color emojiReactionActiveBackgroundHovered
+    }
```

Comment:
These aren't related to this PR, are they?

Reply:
Yeah, those are in a separate commit, as noted in the PR description

Reply:
Ha, next time I should read your PR description